docs/mxnet_backend/save_mxnet_model.md

```diff
@@ -38,18 +47,45 @@ model.fit(x_train, y_train,
           epochs=epochs,
           verbose=1,
           validation_data=(x_test, y_test))
-          
+
 # Save the trained Keras Model as MXNet Model
 keras.models.save_mxnet_model(model=model, prefix='my_model')
 
 # You get the MXNet model - (my_model-symbol.json, my_model-0000.params) in your current directory.
 # Symbol and Params are two files representing a native MXNet model.
 
 ```
- 
+
+## Checkpoint MXNet model using MXNetModelCheckpoint callback
+
+Using `MXNetModelCheckpoint` is similar to using Keras `ModelCheckpoint` callback. Please see here - https://keras.io/callbacks/#modelcheckpoint for callback parameters.
+MXNetModelCheckpoint has following differences compared to ModelCheckpoint:
+1. Accepts one additional parameter - `prefix` - name of the saved Model (symbol and params) files. Model will be saved as 'prefix-symbol.json' and 'prefix-epoch.params'.
+2. Model checkpoint are always saved in the current working directory i.e., MXNetModelCheckpoint do not accept `filepath`. (This will be supported soon - [#131](https://github.com/awslabs/keras-apache-mxnet/issues/131))
+
+You can use MXNetModelCheckpoint to checkpoint(save) the model as native MXNet model based on your checkpoint criteria - saving the best model, saving model after each epoch etc.
+
+To obtain MXNet Model details for later binding to MXNet Module, you can use `K.get_mxnet_model_info()` API that returns `data_names` and `data_shapes` that you can use to bind the model with MXNet module.
+
+```python
+# ... Assuming you have built the model.
+
+# Use MXNetModelCheckpoint callback to save best model during the training.
+checkpoint = MXNetModelCheckpoint(mxnet_model_prefix="my_model", monitor = 'val_loss', verbose = 1, save_best_only = True, mode = 'min')
+callbacks_list = [checkpoint]
+model.compile(loss=keras.losses.categorical_crossentropy,
+              optimizer=keras.optimizers.Adadelta(),
+              metrics=['accuracy'], callbacks=callbacks_list)
+
+# You get the best MXNet model - (my_model-symbol.json, my_model-0000.params) in your current directory.
+# Symbol and Params are two files representing a native MXNet model.
+```
+
```

Comment:
should we explain modes also here?

Reply:
@kalyc  do you mean `mode = 'min'` argument? it's explained in the docstring, and it's the same cross other Keras checkpoints. I think it's fine here.